spec/xcpretty/parser_spec.rb

```diff
@@ -84,6 +84,11 @@ module XCPretty
       @parser.parse(SAMPLE_ANOTHER_COMPILE.sub('.m', '.mm'))
     end
 
+    it 'parses compiling Swift source files' do
+      @formatter.should receive(:format_compile).with("KWNull.swift", "Classes/Core/KWNull.swift")
```

Comment:
Line is too long. [98/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.